### DISCUSSION TITLE
Self-nomination to Go Contributors team

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#121] [Go Team]
Hi @paketo-buildpacks/go-maintainers - I would like to self-nominate for the go contributors team.

I have made a variety of contributions - from small refactors and tidying up loose ends to exploring what SBOM and jammy tiny stack support could look like in the go ecosystem. I have also reviewed multiple PRs of various sizes and I shared my perspective on issues of varying complexity. I intend to continue these contributions for the foreseeable future and becoming a contributor will enable me to do so more effectively.

Thank you,
Rob

**Comment by robdimsdale** [comment]
@paketo-buildpacks/steering-committee looks we have a supermajority in favor of me becoming a member of the golang contributors team.

**Comment by dmikusa** [comment]
Congrats! I've added you to the @paketo-buildpacks/go-contributors team. I'll submit a PR to the community repo in a min.
